### PR TITLE
Issue206 mixing volume

### LINKED ISSUE #206: overspecified initial conditions in examples with mixing volume
The models
```
simulateModel("Annex60.Fluid.MixingVolumes.Examples.MixingVolumeMassFlow", stopTime=10, method="dassl", resultFile="MixingVolumeMassFlow");
```
and
```
simulateModel("Annex60.Fluid.MixingVolumes.Examples.MixingVolumeMFactor", method="dassl", stopTime=100, resultFile="MixingVolumeMFactor");
```
give the warnings below in Dymola 2015 FD01:

```
Translation of Annex60.Fluid.MixingVolumes.Examples.MixingVolumeMassFlow:

The DAE has 112 scalar unknowns and 112 scalar equations.

Redundant consistent initial conditions.

Removed the following equations which are redundant and consistent: 
if (vol.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics.FixedInitial) then 
if (vol.dynBal.initialize_p) then 
vol.dynBal.medium.p = vol.dynBal.p_start;
end if;
else
if (vol.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics. SteadyStateInitial) then 
if (vol.dynBal.initialize_p) then 
der(vol.dynBal.medium.p) = 0;
end if;
end if;
end if;

This required evaluation of 
the interactive parameters 
vol.p_start(start = 101325)

The redundancies involve
initial equations: 
vol.dynBal.medium.p = vol.dynBal.p_start;

```
and
```
Translation of Annex60.Fluid.MixingVolumes.Examples.MixingVolumeMFactor:

The DAE has 313 scalar unknowns and 313 scalar equations.

Redundant consistent initial conditions.

Removed the following equations which are redundant and consistent: 
if (vol.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics.FixedInitial) then 
if (vol.dynBal.initialize_p) then 
vol.dynBal.medium.p = vol.dynBal.p_start;
end if;
else
if (vol.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics. SteadyStateInitial) then 
if (vol.dynBal.initialize_p) then 
der(vol.dynBal.medium.p) = 0;
end if;
end if;
end if;
if (volMFactor.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics. FixedInitial) then 
if (volMFactor.dynBal.initialize_p) then 
volMFactor.dynBal.medium.p = volMFactor.dynBal.p_start;
end if;
else
if (volMFactor.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics. SteadyStateInitial) then 
if (volMFactor.dynBal.initialize_p) then 
der(volMFactor.dynBal.medium.p) = 0;
end if;
end if;
end if;
if (vol1.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics.FixedInitial) then 
if (vol1.dynBal.initialize_p) then 
vol1.dynBal.medium.p = vol1.dynBal.p_start;
end if;
else
if (vol1.dynBal.massDynamics == Modelica.Fluid.Types.Dynamics. SteadyStateInitial) then 
if (vol1.dynBal.initialize_p) then 
der(vol1.dynBal.medium.p) = 0;
end if;
end if;
end if;

This required evaluation of 
the interactive parameters 
vol.p_start(start = 101325)
vol1.p_start(start = 101325)
volMFactor.p_start(start = 101325)

The redundancies involve
initial equations: 
vol.dynBal.medium.p = vol.dynBal.p_start;
volMFactor.dynBal.medium.p = volMFactor.dynBal.p_start;
vol1.dynBal.medium.p = vol1.dynBal.p_start;
```

In Dymola 2016 beta 2, the model `MixingVolumeMassFlow` does not translate:
```
The initialization problem is overspecified for variables of element type Real.
The initial equation 
vol.dynBal.medium.p = vol.dynBal.p_start;
refers to variables, which all are known.
To correct it you can remove this equation
```
This can be fixed by adding a resistance.

### PR DESCRIPTION
**Pull request by mwetter**
This will close #206

### CODE REVIEW

**Comment by Mathadon** [comment]
In my opinion it would be better to leave out the resistances and set ```massDynamics``` to ```SteadyState```. This also solves the problem and it does not add complexity to the problem. Therefore it can serve as an example for users that encounter a similar problem.

File: Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMFactor.mo [hunk 1]
```diff
@@ -73,6 +93,13 @@ difference between using an <code>mSenFac = 10</code> and multiplying volume by
 </html>", revisions="<html>
 <ul>
 <li>
+March 26, 2015 by Michael Wetter:<br/>
+Added resistance between the volume and the pressure boundary
+condition to avoid an overspecified by consistent initial value problem.
```
Comment (Mathadon) [inline Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMFactor.mo]: 'by' can be removed in this sentence

File: Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMassFlow.mo [hunk 2]
```diff
@@ -45,6 +59,13 @@ This model demonstrates the use of the mixing volume with air flowing into and o
 </html>", revisions="<html>
 <ul>
 <li>
+March 26, 2015 by Michael Wetter:<br/>
+Added resistance between the volume and the pressure boundary
+condition to avoid an overspecified by consistent initial value problem.
```
Comment (Mathadon) [inline Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMassFlow.mo]: 'by' can be removed

File: Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMFactor.mo (hunk 1 above)
Comment (mwetter) [inline Annex60/Fluid/MixingVolumes/Examples/MixingVolumeMFactor.mo]: Yes, it should be "but consistent". I will fix it.

**Comment by mwetter** [comment]
@Mathadon 
I agree to your proposed changes. Can you please review again. I did the following:

- Removed the pressure drops and changed the parameter for the mass dynamics to use unspecified initial conditions.
- Updated the info section to explain the rationale for this selection.
- Updated revision section
- Used in `MixingVolumeMFactor` the same instance for the boundary condition for all three volumes as I do not see why a second instance should be used.
- Uploaded new reference results.

**Comment by Mathadon** [comment]
I actually proposed to use ```SteadyState``` massDynamics instead of ```DynamiceFreeInitial```, but if this works too then it's ok for me!

**Pull request merged**